modules/services/window-managers/river.nix

```diff
@@ -0,0 +1,213 @@
+{ config, lib, pkgs, ... }:
+
+with lib;
```

Comment:
This is deprecated. Just inherit only what you need out of it.

Reply:
Done :)

Reply:
Merci